deploy/ansible/roles/k8s_create_cluster/templates/kops.yaml.j2

```diff
@@ -154,8 +154,16 @@ metadata:
     kops.k8s.io/cluster: {{ cluster_name }}
   name: nodes.{{ shape }}
 spec:
+  hooks:
+  - name: resize-nvme-rootfs
+    roles:
+    - Node
+    manifest: |
+      Type=oneshot
+      ExecStart=/bin/sh -c 'test -b /dev/nvme0n1p1 && growpart-workaround /dev/nvme0n1 1 && resize2fs /dev/nvme0n1p1 || true'
```

Comment:
@MaksymBilenko maybe we should avoid hardcoded constants like `/dev/nvme0n1p1` ? It would be better to detect names of disks by tools such as `fdisk -l 2>/dev/null | tail -n 1 | cut -f1 -d" "`

Reply:
@kirillmakhonin At the most cases it does resize well, the issue is that at this moment Kops unable to do this with `nvme` I'm afraid of resizing xvda by 2 separate tasks on node bootstrap.

Reply:
@MaksymBilenko it looks like ugly workaround. Are you sure that this workaround works for all shapes? What if other shapes will be defined for cluster? Could you use shapes which work well with new kops instead of such workaround?